content/_index.en.md

```diff
@@ -1,5 +1,441 @@
+
+### Terms types
+
+To distinguish between the different **terms** of a **service**, each has a **type**, such as “Terms of Service”, “Privacy Policy”, “Developer Agreement”…
+
+This **type** matches the topic, but not necessarily the title the **service** gives to it. Unifying the **types** enables comparing **terms** across **services**.
+
+> More information on terms types can be found in the [dedicated repository](https://github.com/OpenTermsArchive/terms-types). They are published on NPM under [`@opentermsarchive/terms-types`](https://www.npmjs.com/package/@opentermsarchive/terms-types), enabling standardisation and interoperability beyond the Open Terms Archive engine.
+
+### Declarations
+
+The **documents** that constitute a **collection** are defined in simple JSON files called **declarations**.
+
+A **declaration** also contains some metadata on the **service** the **documents** relate to.
+
+> Here is an example declaration tracking the Privacy Policy of Open Terms Archive:
+>
+> ```json
+> {
+>   "name": "Open Terms Archive",
+>   "documents": {
+>     "Privacy Policy": {
+>       "fetch": "https://opentermsarchive.org/en/privacy-policy",
+>       "select": ".TextContent_textContent__ToW2S"
+>     }
+>   }
+> }
+> ```
+
+## How to add documents to a collection
+
+Open Terms Archive **acquires** **documents** to deliver an explorable **history** of **changes**. This can be done in two ways:
+
+1. For the present and future, by **tracking** **documents**.
+2. For the past, by **importing** from an existing **fonds** such as [ToSBack](https://tosback.org), the [Internet Archive](https://archive.org/web/), [Common Crawl](https://commoncrawl.org) or any other in-house format.
+
+### Tracking documents
+
+The **engine** **reads** **declarations** to **record** a **snapshot** by **fetching** the declared web **location** periodically. The **engine** then **extracts** a **version** from this **snapshot** by:
+
+1. **Selecting** the subset of the **snapshot** that contains the **terms** (instead of navigation menus, footers, cookies banners…).
+2. **Removing** residual content in this subset that is not part of the **terms** (ads, illustrative pictures, internal navigation links…).
+3. **Filtering noise** by preventing parts that change frequently from triggering false positives for **changes** (tracker identifiers in links, relative dates…). The **engine** can execute custom **filters** written in JavaScript to that end.
+
+After these steps, if **changes** are spotted in the resulting **document**, a new **version** is **recorded**.
+
+Preserving **snapshots** enables recovering after the fact information potentially lost in the **extraction** step: if **declarations** were wrong, they can be **maintained** and corrected **versions** can be **extracted** from the original **snapshots**.
```

Comment:
Should we mention here the experimental cleaning dataset project? I believe it could be interesting